conbench/app/_plots.py

```diff
@@ -333,11 +467,27 @@ def time_series_plot(history, benchmark, run, height=380, width=1100):
         x_axis_type="datetime",
         height=height,
         width=width,
-        tools=["pan", "zoom_in", "zoom_out", "reset"],
+        tools=["pan", "zoom_in", "zoom_out", "reset", "tap"],
         x_range=(t_start, t_end),
     )
     p.toolbar.logo = None
 
+    # TapTool is not responding to each click event, but but only triggers when
+    # clicking a glyph:
+    # https://discourse.bokeh.org/t/how-to-trigger-callbacks-on-mouse-click-for-tap-tool/1630
+    taptool = p.select(type=bokeh.models.TapTool)
+    taptool.callback = gen_js_callback_click_on_glyph_show_run_details(
+        # Assume that the repository is constant across all data points in
+        # this plot. Is that a good assumption
```

Comment:
I'd love to have feedback on this! 

Do all samples in a history plot have the same repository specified in their contexts? Any guarantee on that? CC @austin3dickey

Reply:
Thanks, especially for linking to it! :)

Reply:
updated comment